build-aux/snap/snapcraft.yaml

```diff
@@ -203,10 +203,23 @@ parts:
       craftctl default
       # set version, this needs dpkg-parsechangelog (from dpkg-dev) and git
       VERSION="$(./mkversion.sh --output-only)"
-      if [ -f fips-build ] ; then
-          echo "-- appending FIPS tag to version $VERSION"
-          VERSION="$VERSION-fips"
+      if [ "${VERSION%/+fips/}" != "$VERSION" ] ; then
+          # we have a '+fips' element in the version, which may be coming from
+          # debian/changelog or git tag
+          echo "-- deteceted FIPS build"
+          touch fips-build
       fi
+      # TODO detect when doing a FIPS snap build on LP
```

Comment:
Is there a ticket for that?

Reply:
yup https://warthogs.atlassian.net/browse/SNAPDENG-23124